user-guide/Advanced_Modules/DOM/DOM_objects/DOM_FieldDescriptors/DOM_AutoIncrementFieldDescriptor.md

```diff
@@ -0,0 +1,40 @@
+---
+uid: DOM_AutoIncrementFieldDescriptor
+---
+
+# AutoIncrementFieldDescriptor
+
+- **FieldValue type**: string
+- **Multiple values supported**: :heavy_multiplication_x:
+
+| Type of Descriptor | FieldType | FieldValue type |
+|--------------------|-----------|-----------------|
+| AutoIncrement | string | string |
+
+Defines a field that will automatically get an incremented value assigned. When a DomInstance does not have a value for this field yet, it will get assigned the next time the instance is updated.
+
+The `IDFormat` property is used to define a string format for the number value. In this string, "{0}" gets replaced by that number value. If the `IDFormat` property is empty, its value will not be formatted.
+
+When the field is marked as soft-deleted, no values will get assigned when a `DomInstance` gets saved.
+
+Some examples, assuming the next value is 10:
+
+| Description | Format | Result |
```

Comment:
This is duplicated from the 'SectionDefinition' page, do we need this information in both places?